src/app/services/audio-request/audio-request.service.ts

```diff
@@ -41,10 +41,9 @@ export class AudioRequestService {
 
   getAudioRequests(expired: boolean): Observable<UserAudioRequest[]> {
     return this.userProfile$.pipe(
```

Comment:
You can remove the wrapping `this.userProfile$.pipe()` as it was only used to get the user_id

Reply:
Removed from here and also `getUnreadCount()`